tests/js-modules/modules.py

```diff
@@ -948,6 +947,20 @@ def test_npm_app(network, args):
             body = r.body.json()
             assert body["claims"]["unique_id"] == primary_quote_info["mrenclave"], body
             assert "sgx_report_data" in body["customClaims"], body
+        elif args.enclave_platform == "snp":
+            LOG.info("SNP: Test verifySnpAttestation")
+            r = c.post(
+                "/app/verifySnpAttestation",
+                {
+                    "evidence": primary_quote_info["raw"],
+                    "endorsements": primary_quote_info["endorsements"],
+                    "endorsed_tcb": primary_quote_info["endorsed_tcb"],
+                },
+            )
+            assert r.status_code == http.HTTPStatus.OK, r.status_code
```

Comment:
It would be good to test this a bit more: evidence too big, too short, endorsements that don't match, endorsed_tcb that doesn't match etc.

Reply:
https://github.com/microsoft/CCF/pull/5653/commits/ff9aefa7538755b0d1fa8de035f490d39ab90523